spec/Section 5 -- Validation.md

```diff
@@ -570,6 +570,50 @@ fragment conflictingDifferingResponses on Pet {
 }
 ```
 
+Fragment arguments can also cause fields to fail to merge.
+
+While the following is valid:
+
+```graphql example
+fragment commandFragment($command: DogCommand!) on Dog {
+  doesKnowCommand(dogCommand: $command)
+}
+
+fragment potentiallyConflictingArguments(
+  $commandOne: DogCommand!
+  $commandTwo: DogCommand!
+) on Dog {
+  ...commandFragment(command: $commandOne)
+  ...commandFragment(command: $commandTwo)
+}
+
+fragment safeFragmentArguments on Dog {
+  ...potentiallyConflictingArguments(commandOne: SIT, commandTwo: SIT)
+}
+```
+
+it is only valid because `safeFragmentArguments` uses
+`potentiallyConflictingArguments` with the same value for `commandOne` and
+`commandTwo`. Therefore `commandFragment` resolves `doesKnowCommand`'s
+`dogCommand:` arg to `SIT` in both cases.
+
+However, by changing the argument values:
+
+```graphql counter-example
+fragment conflictingFragmentArguments on Dog {
+  ...potentiallyConflictingArguments(commandOne: SIT, commandTwo: DOWN)
+}
+```
+
+the response will have two conflicting versions of the `doesKnowCommand` that
+cannot merge.
+
+One strategy to resolving field conflicts caused by duplicated fragment spreads
+is to short-circuit when two fragment spreads with the same name are found to be
```

Comment:
not sure what short-circuit means

Reply:
I'm going to update the algorithm to explicitly fail if two spreads with the same name have different arguments, rather than using a "short circuit" (stop early) option.